daemon/api_sideload_n_try_test.go

```diff
@@ -1093,3 +1093,30 @@ func (s *trySuite) TestTryChangeConflict(c *check.C) {
 	rspe := daemon.TrySnap(st, tryDir, snapstate.Flags{}).(*daemon.APIError)
 	c.Check(rspe.Kind, check.Equals, client.ErrorKindSnapChangeConflict)
 }
+
+func (s *sideloadSuite) TestSideloadSnapInvalidTransaction(c *check.C) {
+	s.daemon(c)
+
+	// try a multipart/form-data upload with missing "name"
```

Comment:
I don't think this is relevant to this test, probably left over from `TestSideloadSnapNotValidFormFile`